packages/react-composites/src/composites/CallComposite/LocalPreview.tsx

```diff
@@ -38,6 +32,20 @@ export const LocalPreview = (): JSX.Element => {
     isLocalMicrophoneEnabled ? adapter.mute() : adapter.unmute();
   }, [adapter, isLocalMicrophoneEnabled]);
 
+  const theme = useTheme();
+  const onRenderPlaceholder = useCallback((): JSX.Element => {
+    return (
+      <Stack style={{ width: '100%', height: '100%', color: theme.palette.black }} verticalAlign="center">
```

Comment:
nit: this should use `styles` or `className` for setting styling on a fluent element (using styles or className creates a css class for the element, setting `style` directly sets the style on the dom element inline)

Reply:
was it intentional this changed from `theme.palette.neutralPrimaryAlt` to `theme.palette.black` ?

Reply:
From the figma looks like our other icons are `palette.neutralPrimary` (not black or alt): https://www.figma.com/file/72qJ03EOQExf12g5JTdWjQ/ACS-UI---POR?node-id=2136%3A8597

Reply:
Good catch. I will change it to `palette.neutralPrimary`.